cdk/bin/cdk.ts

```diff
@@ -4,4 +4,40 @@ import { Frontend } from "../lib/frontend";
 
 const app = new App();
 const cloudFormationStackName = process.env.GU_CFN_STACK_NAME;
-new Frontend(app, "Frontend", { stack: "support", cloudFormationStackName });
+
+new Frontend(app, "Frontend-PROD", {
+  stack: "support",
+  stage: "PROD",
+  cloudFormationStackName,
+  membershipSubPromotionsTable:
+    "arn:aws:dynamodb:*:*:table/MembershipSub-Promotions-PROD",
+  redemptionCodesTable: "arn:aws:dynamodb:*:*:table/redemption-codes-PROD",
+  certificateProps: {
+    domainName: "support.theguardian.com.origin.membership.guardianapis.com",
+    hostedZoneId: "Z1E4V12LQGXFEC",
```

Comment:
As this value is static for each stage, we could probably have it in the `lib/frontend.ts` once.

Reply:
Good spot! I'll factor that out now 👍